### DISCUSSION TITLE
[vscode插件]专为@formily/react 提供的表达式语法支持插件，欢迎大家使用！

### DISCUSSION THREAD

**Discussion by moushicheng** [alibaba/formily#4074] [Show and tell]
vscode插件市场搜索formily-exporession-support即可查询
该插件支持能力如下⬇️
<img width="975" alt="image" src="https://github.com/alibaba/formily/assets/54257412/d800a8ce-9ed6-472f-a695-c1765ffbdb78">
仓库地址:
https://github.com/moushicheng/formily-expression-support

**Comment by zxq08** [comment]
大佬名字是不是写错了，formily-exp~~o~~ression-support
